messaging-experiments.json

```diff
@@ -1146,5 +1146,488 @@
+      "branches": [
+        {
+          "slug": "control",
+          "ratio": 1,
+          "value": {},
+          "groups": [
+            "cfr"
+          ]
+        },
+        {
+          "slug": "treatment-a",
+          "ratio": 1,
+          "groups": [
+            "cfr"
+          ],
+          "value": {
+            "id": "PIP_CFR_NEWS",
+            "groups": [
+              "cfr"
+            ],
+            "content": {
+              "bucket_id": "PIP_CFR_NEWS",
+              "buttons": {
+                "primary": {
+                  "action": {
+                    "data": {
+                      "args": "https://support.mozilla.org/en-US/kb/about-picture-picture-firefox",
+                      "where": "tabshifted"
+                    },
+                    "type": "OPEN_URL"
+                  },
+                  "label": {
+                    "value": "Learn More",
+                    "attributes": {
+                      "accesskey": "L"
+                    }
+                  }
+                },
+                "secondary": [
+                  {
+                    "action": {
+                      "type": "CANCEL"
+                    },
+                    "label": {
+                      "value": "Remind Me Later",
+                      "attributes": {
+                        "accesskey": "R"
+                      }
+                    }
+                  },
+                  {
+                    "label": {
+                      "string_id": "cfr-doorhanger-extension-never-show-recommendation"
+                    }
+                  },
+                  {
+                    "action": {
+                      "data": {
+                        "category": "general-cfrfeatures"
+                      },
+                      "type": "OPEN_PREFERENCES_PAGE"
+                    },
+                    "label": {
+                      "string_id": "cfr-doorhanger-extension-manage-settings-button"
+                    }
+                  }
+                ]
+              },
+              "category": "cfrFeatures",
+              "heading_text": "Watch videos while you browse",
+              "icon": "chrome://global/skin/media/pictureinpicture.svg",
+              "info_icon": {
+                "label": {
+                  "string_id": "cfr-doorhanger-extension-sumo-link"
+                },
+                "sumo_path": "extensionrecommendations"
+              },
+              "layout": "icon_and_message",
+              "notification_text": {
+                "string_id": "cfr-doorhanger-extension-notification"
+              },
+              "text": "Keep yourself in the loop. With Picture-in-Picture you can watch the latest news while you visit other websites or apps - hover over a video and select the blue icon."
+            },
+            "frequency": {
+              "lifetime": 30,
```

Comment:
Is this correct? We want to show the message 30 times? The usual lifetime is `3`.

Reply:
Confirmed with experiment owners that we're good to use the lifetime as 3.